web_app/database/models.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime
+from uuid import uuid4
+
+from sqlalchemy import UUID, Column, DateTime, Float, String
+from sqlalchemy_utils import IPAddressType
+
+from .database import Base
+
+
+class NotificationData(Base):
+    __tablename__ = "notification"
+
+    id = Column(UUID, default=uuid4, primary_key=True)
+    created_at = Column(DateTime, default=datetime.now())
+    email = Column(String, index=True, unique=True, nullable=False)
+    wallet_id = Column(String, nullable=False)
+    telegram_id = Column(String, unique=True, nullable=False)
+    ip_address = Column(IPAddressType, nullable=False)
+    health_ratio_level = Column(Float, nullable=False)
```

Comment:
add please `protocol_id` String type

Reply:
Please, add also choice list as Enum for protocol_id field
`["Hashstack", "Nostra", "zkLend"]`